site/resources.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: site
+  labels:
+    app: site
+spec:
+  selector:
+    matchLabels:
+      app: site
+  template:
+    metadata:
+      labels:
+        app: site
+    spec:
+      containers:
+      - name: site
+        image: hashbang/site
+        ports:
+        - name: http
+          containerPort: 8080
+          protocol: TCP
+        livenessProbe:
```

Comment:
might want to copy the old settings? https://github.com/hashbang/infra/blob/8b80cfb64406f4f36fc80b10b6968be1132e227e/charts/site/templates/deployment.yaml#L37-L54